model/_global/create_pit/structures.smithy

```diff
@@ -37,6 +37,6 @@ structure CreatePit_Input with [CreatePit_QueryParams] {
 @output
 structure CreatePit_Output {
     pit_id: String,
-    _shard: ShardStatistics,
+    _shards: ShardStatistics,
```

Comment:
@Xtansia Does this change need regenerating whatever we generated for the python client?

Reply:
This is part of the response body, so won't have been used, just noticed it while writing bodies for .NET